2.train-VAE/utils/betatcvae.py

```diff
@@ -291,7 +291,7 @@ def compile_tc_vae(model, train_loader, val_loader, test_loader, optimizer, epoc
 
     return train_loss_history, val_loss_history, test_loss_history
 
-def extract_latent_dimensions(model, data_loader, metadata, path):
+def tc_extract_latent_dimensions(model, data_loader, metadata, path):
```

Comment:
Consider adding type hints for the parameters and return in this function. Also, it looks like the docstring below doesn't include `path`. An example of type hints (just making a guess based on the docstring):
```suggestion
def tc_extract_latent_dimensions(model: VAE, data_loader: DataLoader, metadata: pd.DataFrame, path: pathlib.Path) -> pd.DataFrame:
```

Reply:
Done, thank you!